.github/scripts/prepare_production_deployment.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -ev
+
+# Only:
+# - Tagged commits
+# - Security env variables are available.
+if [ -n "$VERSION_TAG" ] && [ -n "$PROD_DEPLOYMENT_HOOK_TOKEN" ] && [ -n "$PROD_DEPLOYMENT_HOOK_URL" ]
+then
+  curl --silent --output /dev/null --write-out "%{http_code}" -X POST \
```

Comment:
Where does `"%{http_code}"` come from?
Oh, that's from the response from the POST, right?

Reply:
You're right, it is the http code from the POST response, for debugging purposes